builtin/providers/google/resource_dns_managed_zone.go

```diff
@@ -30,7 +30,7 @@ func resourceDnsManagedZone() *schema.Resource {
 
 			"description": &schema.Schema{
 				Type:     schema.TypeString,
-				Optional: true,
+				Required: true,
```

Comment:
This is generally our pattern to leave the field optional and fill it with "Managed by Terraform" to minimize required boilerplate for Terraform users.